spec/support/database.yml

```diff
@@ -0,0 +1,14 @@
+default_env:
```

Comment:
Hmm, is this necessary because we don't set `RAILS_ENV` anywhere?

Reply:
Yes, and that’s because I run this spec in isolation (to have a separate database schema). Will set it manually here.